README.md

```diff
@@ -365,6 +365,49 @@ private Binding<HttpStatus> problems(final Condition<HttpStatus> condition) {
 }
 ```
 
+#### Template Routing
+
+Routing and response handling is sometimes pretty similar for many request, and can even be handle using the same
+defined routing logic for different requests. To enable reuse of this routing logic, Riptide supports the creation
+of nested *template routing*, that can be reused for dispatching different request. The following example defines
+a nested hierarchy of routing templates:
+
+```java
+static final Router<MediaType> CONTENT_ROUTER = Router.create(contentType(),
```

Comment:
No, it is not. It is stated on the idea, that definitions can be much smaller, if you once define the routing template and than change/reuse it for many different requests points.